bob-api/index.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+// Include the api
```

Comment:
Deze comment is niet nodig

Reply:
In deze file heb je een aantal keer dezelfde opmerking gemaakt, namelijk 'onnodige comments'. In zulke gevallen is het vaak handiger om 1 keer globaal in het bestand aan te geven: 'ik denk dat er veel onnodige comments in dit bestand staan, kan je daar nog een keer naar kijken, of zullen we dat even samen doen?'. Dan komt het een stuk minder 'zeurderig' over. 

Verder heb je helemaal gelijk dat alle comments overbodig zijn ;)